src/gatling/scala/IDEPathHelper.scala

```diff
@@ -2,7 +2,7 @@ import java.nio.file.{Path, Paths}
 
 object IDEPathHelper {
 
-  private val projectRootDir = Paths.get(getClass.getClassLoader.getResource("gatling.conf").toURI()).getParent.getParent.getParent
+  private val projectRootDir = Paths.get(getClass.getClassLoader.getResource("gatling.conf").toURI()).getParent.getParent.getParent.getParent
```

Comment:
We were resolving the `build` directory instead of the actual root... The Java and Kotlin demos do not have this bug.